src/mongo/db/pipeline/expression.cpp

```diff
@@ -3144,8 +3144,18 @@ Value ExpressionPow::evaluate(const Document& root) const {
     }
 
     long long result = 1;
+
+    // When baseLong == -1 and expLong is < 0 the following for loop will never run because expLong
```

Comment:
We tend to put variable names in single quotes within comments, to make it very obvious you're referring to the variable. Here it's not confusing, but sometimes variables have generic names like 'input' or 'it'.

So, here the comment should look like:
`// When 'baseLong' is -1 and 'expLong' is negative, the following for loop will never run because 'expLong' ...`